keyboards/rgbkb/zen/rev1/keymaps/samae/keymap.c

```diff
@@ -0,0 +1,89 @@
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+enum layer_number {
+    _QWERTY = 0,
```

Comment:
You don't need 0 explicitly defined here. But either way is just fine. ```suggestion
    _QWERTY,
```

Reply:
I'm not bothered too much by being explicit here. If it's ok with  you I'd rather keep it.